tests/unit/lms/validation/_canvas_test.py

```diff
@@ -109,105 +205,20 @@ def test_it_raises_ValidationError_if_one_of_the_objects_in_the_response_json_ha
         assert exc_info.value.messages == {0: {"_schema": ["Invalid input type."]}}
```

Comment:
You mean it could just modify one of the existing dicts in `schema.context["response"].json.return_value`, instead of replacing `return_value` wholesale? I think that would be better yes